CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ the `Marshaler<Writer>` transform overload that were _mistakenly_ passing
 `Writer` explicitly (e.g. `bond::Apply<Writer>(marshaler, value)`) will now
 get a compiler error. To fix, remove the `<Writer>` part:
 `bond::Apply(marshaler, value)`.
+* C++ codegen now generates [extern templates](http://en.cppreference.com/w/cpp/language/function_template) of `bond::Apply` instead of overloads.
```

Comment:
Is this technically a breaking change since we are changing the namespace of the Apply functions?